.github/workflows/ping-other-repos.yml

```
@@ -0,0 +1,19 @@
+name: Ping other repos
+
+on:
+  push:
+    branches: [main]
+  repository_dispatch:
+
+jobs:
+  ping-pong:
```

I guess we're no longer `pong`-ing now, too

```suggestion
  ping:
```